src/editors/textEditor.js

```diff
@@ -73,6 +76,13 @@ class TextEditor extends BaseEditor {
      * @type {CSSStyleDeclaration}
      */
     this.textareaParentStyle = void 0;
+    /**
+     * z-index class style for the editor.
+     *
+     * @private
+     * @type {string}
+     */
+    this.zIndexClass = void 0;
```

Comment:
I would call this variable as `layerClass`. For me, it would better call the being, not just it's feature.